calico_containers/calicoctl.py

```diff
@@ -437,6 +461,64 @@ def normalize_version(version):
     """
     return [int(x) for x in re.sub(r'(\.0+)*$','', version).split(".")]
 
+
+def get_host_ips():
```

Comment:
Looking for advice on whether to use regex or split the output by spaces and endlines (see get_host_ip6s). The ipv6 regex is a bit rediculous, with the double colon shortener.